apps/subgraph/.eslintrc.js

```diff
@@ -18,6 +18,7 @@ module.exports = {
     warnOnUnsupportedTypeScriptVersion: true,
   },
   rules: {
+    "@typescript-eslint/consistent-type-imports": "off",
```

Comment:
Why make this change?

Reply:
It's because subgraph has AssemblyScript and it doesn't support type import.